CHANGELOG.md

```diff
@@ -26,9 +27,99 @@ Packages are formatted by release version. You can see all the changes done to p
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html) as explained in the [Backward Compatibility Promise](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/).
 
-The changelog is generated during the release process using [ChangelogLinker](https://github.com/symplify/changeloglinker) since `v7.0.0-alpha6` release.
-
-<!-- changelog-linker -->
+The changelog is generated during the release process using [Github Changelog Generator](https://github.com/github-changelog-generator/github-changelog-generator) since `v9.1.1` release.
+
+## [v9.1.1](https://github.com/shopsys/shopsys/tree/v9.1.1) (2021-04-14)
```

Comment:
Just thinking about the link here. In the previous releases, it was "full changelog" link. 
Now it's a source code in a specific tag. I thought about linking the release on Github, but there is just a link to changelog. 

Do you find it beneficial to have a link to the source code and the next link is the full changelog?

Reply:
I have updated it to previous style